### PR TITLE
Improving the DEEPWALK example.

### PR DESCRIPTION
**Pull request by drivanov**
## Description

In the current PR we eliminate the following warning:
```
/usr/local/lib/python3.10/dist-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
```
We have also changed the output of information about training time for batches. Now this information is displayed on the same line of the screen for all batches.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [x] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: examples/pytorch/ogb/deepwalk/deepwalk.py [hunk 1]
```diff
@@ -247,26 +247,29 @@ def fast_train(self):
                 if i > 0 and i % self.args.print_interval == 0:
                     if self.args.print_loss:
                         print(
-                            "Batch %d training time: %.2fs loss: %.4f"
+                            "\rBatch %d training time: %.2fs loss: %.4f"
```
Comment (frozenbugs) [inline examples/pytorch/ogb/deepwalk/deepwalk.py:250]: why this change?

Comment (frozenbugs) [reply examples/pytorch/ogb/deepwalk/deepwalk.py:250]: revert?

Comment (drivanov) [reply examples/pytorch/ogb/deepwalk/deepwalk.py:250]: Without this change, we do have more than 1100 lines (1138, to be exact) similar to the following one:
```
Batch 93700, training time: 1.49s
```
I think it would be better to just output this information to the same line of the screen and change this line about every 1.5 seconds.

Comment (frozenbugs) [reply examples/pytorch/ogb/deepwalk/deepwalk.py:250]: Usually for example in DGL, we output newline without erase the previous. And we use i % self.args.print_interval = 0 to reduce the frequency if needed.

Comment (drivanov) [reply examples/pytorch/ogb/deepwalk/deepwalk.py:250]: Reverted

**Comment by frozenbugs** [comment]
Thanks for taking time to make DGL example code excellent!
